keyboards/splitkb/kyria/keymaps/vial/rules.mk

```
@@ -0,0 +1,46 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no # Mouse keys
+EXTRAKEY_ENABLE = yes      # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no          # Audio output
+SPLIT_KEYBOARD = yes       # Split common
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
+
+DEFAULT_FOLDER = splitkb/kyria/rev2
+
```

```suggestion
ENCODER_ENABLE = yes
ENCODER_MAP_ENABLE = yes
```
Most this stuff is replicated from the keyboard level.
The Vial keymap in the main repo shouldn't overwrite the bootloader.

---

I have deleted some options that are identical to the keyboard level. Some are left here to reduce the firmware size